src/components/Card/Card.js

```diff
@@ -2,35 +2,53 @@ import PropTypes from 'prop-types';
 import React from 'react';
 import './Card.css';
 
-const Card = ({ location, districtData, select, id, selected}) => {
-  const years = Object.keys(districtData).map((year, index) => {
-    let style = districtData[year] >= 0.5 ? 'high' : 'low';
-    return (
-      <li className={style}  key={`year-${index}`}>
-        {' '}
-        {year}: {districtData[year]}
-      </li>
-    );
-  });
+const Card = (props) => {
+  let renderedCard;
+  const {isSelected, onSelect} = props;
+  const klass = isSelected ? 'card selected' : 'card';
 
-  const handleCardClick = () => {
-    select(location);
-  };
+  if (!props.compared) {
+    const {location, districtData, id} = props;
+    const years = Object.keys(districtData).map((year, index) => {
+      const style = districtData[year] >= 0.5 ? 'high' : 'low';
+      return (
+        <li className={style}  key={`year-${index}`}>
+          {' '}
+          {year}: {districtData[year]}
+        </li>
+      );
+    });
+    
+    renderedCard = 
+      (<div id={id} className={klass} onClick={() => {onSelect(location, id, isSelected)}}>
+        <h3>{location}</h3>
+        <ul>{years}</ul>
+      </div>)
+  } else {
+    const districtNames = Object.keys(props);
+
+    renderedCard = (
+      <div className={klass}>
+      <h3>{districtNames[1]}</h3>
+      <h4>-- {props.compared} --</h4>
```

Comment:
This was only to make it look like the comp. We will probably end up taking these out because i found some icons we can put in as arrows.